operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -179,6 +179,7 @@ mimir:
       {{- if .Values.query_scheduler.enabled }}
       scheduler_address: {{ template "mimir.fullname" . }}-query-scheduler-headless.{{ .Release.Namespace }}.svc:{{ include "mimir.serverGrpcListenPort" . }}
       {{- end }}
+      parallelize_shardable_queries: true
```

Comment:
should go before the scheduler_address to have same alphabetic order as in the generated config